spec/features/dashboards/show_spec.rb

```diff
@@ -1,79 +1,80 @@
-  # let!(:transaction_11) { invoice_9.transactions.create!(credit_card_number: "4203696133194408", credit_card_expiration_date: "12/26", result: "success") }
-  # let!(:transaction_12) { invoice_10.transactions.create!(credit_card_number: "4801647818676136", credit_card_expiration_date: "09/25", result: "success") }
-  # let!(:transaction_13) { invoice_11.transactions.create!(credit_card_number: "4540842003561938", credit_card_expiration_date: "01/26", result: "failed") }
+        expect(page).to have_content(merchant.name)
+      end
 
-  describe "As a merchant" do
-    it "shows the name of my merchant" do
-      visit merchant_dashboard_path(merchant_id: merchant.id)
+      it "shows link to my merchant item index and merchant invoices index" do
+        visit merchant_dashboard_path(merchant_id: merchant.id)
 
-      expect(page).to have_content(merchant.name)
-    end
+        expect(page).to have_link("My Items", href: merchant_items_path(merchant))
+        expect(page).to have_link("My Invoices", href: merchant_invoices_path(merchant))
+      end
 
-    it "shows link to my merchant item index and merchant invoices index" do
-      visit merchant_dashboard_path(merchant_id: merchant.id)
+      it "shows the names of the top 5 customers and the number of successful transactions with my merchant" do
+        visit merchant_dashboard_path(merchant_id: merchant.id)
 
-      expect(page).to have_link("My Items", href: merchant_items_path(merchant))
-      expect(page).to have_link("My Invoices", href: merchant_invoices_path(merchant))
+        top_customers.each do |top_customer|
+          expect(page).to have_content("#{top_customer.first_name} #{top_customer.last_name} #{top_customer.transactions.count}")
+        end
+      end
     end
 
-    it "shows the names of the top 5 customers and the number of successful transactions with my merchant" do
-      visit merchant_dashboard_path(merchant_id: merchant.id)
+    describe "User Story 4-5" do
+      let(:merchant) { FactoryBot.create(:merchant) }
+      let(:item_1) { FactoryBot.create(:item, merchant: merchant) }
+      let(:item_2) { FactoryBot.create(:item, merchant: merchant) }
+      let(:item_3) { FactoryBot.create(:item, merchant: merchant) }
+      let(:customer) { FactoryBot.create(:customer) }
+
+      let(:invoice_1) { FactoryBot.create(:invoice, customer: customer, created_at: Time.current - 3.day) }
+      let(:invoice_2) { FactoryBot.create(:invoice, customer: customer, created_at: Time.current - 2.day) }
+      let(:invoice_3) { FactoryBot.create(:invoice, customer: customer, created_at: Time.current - 5.day) }
+
+      let!(:invoice_item_1) { InvoiceItem.create!(invoice: invoice_1, item: item_1, status: "packaged") }
+      let!(:invoice_item_2) { InvoiceItem.create!(invoice: invoice_2, item: item_2, status: "packaged") }
+      let!(:invoice_item_3) { InvoiceItem.create!(invoice: invoice_3, item: item_3, status: "shipped") }
```

Comment:
once the enums are created will the test data have to be changed?

Reply:
nope, the enum wont change anything